components/director/internal/domain/bundlereferences/repository_test.go

```diff
@@ -537,24 +530,6 @@ func TestPgRepository_ListAllForBundle(t *testing.T) {
 		sqlMock.AssertExpectations(t)
 	})
 
-	t.Run("returns error when check for internal scope fails", func(t *testing.T) {
```

Comment:
Maybe you can retrun the tests but with the opposite asserts. Something along the lines of:

```
returns both public and internal/private APIs/events when check for internal scope fails
```